### ISSUE TITLE
The path to the extension's directory is wrong

### ISSUE THREAD

**Issue by yutkat** [weirongxu/coc-floatinput#2]
When I install coc-floatinput, I get the following message on startup.

`[coc.nvim] Unable to load global extension at /home/xxx/.config/coc/extensions/coc-floatinput: package.json not found`

`/home/xxx/.config/coc/extensions/node_modules/coc-floatinput` is the correct path.

This is the minimal vimrc.

```vim
syntax enable
filetype plugin indent on

if has('vim_starting')
	let s:pluin_manager_dir='~/.vim/plugged/vim-plug'
	execute 'set runtimepath+=' . s:pluin_manager_dir
endif

call plug#begin('~/.vim/plugged')
Plug 'neoclide/coc.nvim', {'branch': 'release'}
let g:coc_global_extensions = [ 'coc-floatinput' ]
call plug#end()
```

vim: Neovim v0.5.0-695-gf34eeba2d
OS: ArchLinux
coc-floatinput version: 6c2e05ea5ef3962975ac3102ab3f4f477c5b489c
coc.nvim: https://github.com/neoclide/coc.nvim/tree/825f7f5b89fb1708f18711798ef7a2ba6aa1bde5

**Comment by weirongxu** [comment]
Did you try to remove `/home/xxx/.config/coc/extensions/node_modules/coc-floatinput` and install it again? I tried several times in my Manjaro, but no error

**Comment by yutkat** [comment]
Hmmm I'll reproduce it every time.
I recorded this gif.

![Peek 2020-09-16 13-05](https://user-images.githubusercontent.com/8683947/93291237-7a6e2400-f81d-11ea-8b47-be19259d7863.gif)

**Comment by weirongxu** [comment]
It may also be a cache problem, you can try to clear the cache. `npm cache clean --force`

**Comment by yutkat** [comment]
Unfortunately, `npm cache clean --force` didn't solve this problem :cry:

**Comment by weirongxu** [comment]
Okay, provide the version of your node and npm, and the installed log

**Comment by yutkat** [comment]
> node --version
v14.10.1

> npm -v
6.14.7


`:CocOpenLog`

```
2020-09-16T13:52:57.423 ERROR (pid:2647158) [extension:coc-floatinput] - schema.json not found, reinstall coc.nvim to fix this!
2020-09-16T13:52:57.480 INFO (pid:2647158) [plugin] - coc.nvim 0.0.79-b71489207a initialized with node: v14.10.1 after 150ms
2020-09-16T13:53:02.709 ERROR (pid:2647158) [extensions] - Error on load /home/xxx/.config/coc/extensions/coc-floatinput Error: package.json not found
    at Extensions.checkDirectory (/home/xxx/dotfiles/.vim/plugged/coc.nvim/build/index.js:42549:23)
    at /home/xxx/dotfiles/.vim/plugged/coc.nvim/build/index.js:42206:32
    at new Promise (<anonymous>)
    at /home/xxx/dotfiles/.vim/plugged/coc.nvim/build/index.js:42202:79
    at Array.map (<anonymous>)
    at Extensions.globalExtensionStats (/home/xxx/dotfiles/.vim/plugged/coc.nvim/build/index.js:42202:68)
    at Extensions.getExtensionStates (/home/xxx/dotfiles/.vim/plugged/coc.nvim/build/index.js:41922:38)
    at ExtensionList.loadItems (/home/xxx/dotfiles/.vim/plugged/coc.nvim/build/index.js:83060:47)
    at Worker.loadItems (/home/xxx/dotfiles/.vim/plugged/coc.nvim/build/index.js:84737:32)
    at ListManager.start (/home/xxx/dotfiles/.vim/plugged/coc.nvim/build/index.js:77387:31)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async /home/xxx/dotfiles/.vim/plugged/coc.nvim/build/index.js:23614:13
    at async Plugin.cocAction (/home/xxx/dotfiles/.vim/plugged/coc.nvim/build/index.js:23990:16)
    at async NeovimClient.<anonymous> (/home/xxx/dotfiles/.vim/plugged/coc.nvim/build/index.js:11588:21)
```

**Comment by yutkat** [comment]
I found some new information: if you have coc.nvim version v0.0.78, it seems to work fine.

I also built a master and used it, and I had the same problem here.

**Comment by weirongxu** [comment]
Should be fixed in 1.3.2

**Comment by yutkat** [comment]
I checked it. It's fixed. Thank you :wink:
